src/BlockReportSearch/Block/Sql/SqlQuery.php

```diff
@@ -52,12 +52,10 @@ function ($r) {
     private function get_sql_query(Block\Query\Parameters ...$params_list): string
     {
         // Prepare query parameters.
-        $like = [];
```

Comment:
Why did you remove this?

Reply:
So the _Like Class_ is responsible for checking if the `post_content` contains a block that matches a particular type, maybe `core/query` or `planet4-blocks/something`, issue is we don't get back results for the _Actions/Posts List_ blocks since they don't appear in the `post_content`. Since that was the first layer of filtering for `post_ids`, I just removed that line so we always return all the post IDs, then we can later use our own custom filter which we already had, to filter and find the `post_types` with the block being filtered for.